acmwebsite/controllers/root.py

```diff
@@ -59,7 +59,23 @@ def index(self):
         meetings = DBSession.query(Meeting).filter(
             Meeting.date > datetime.datetime.now() - datetime.timedelta(hours=3)
         ).order_by(Meeting.date).limit(2)
-        return dict(page='index', meetings=meetings)
+        try:
+            banner = random.choice(DBSession.query(Banner).all())
+        except:
+            banner = type(
+                '',
+                (object,),
+                dict(
+                    description='Image above: ACM Members and other clubs at a'
+                                'Mozilla Campus Clubs training event',
+                    photo=type(
+                        'obj',
+                        (object,),
+                        {'url': '/img/banner.jpg'}
```

Comment:
This should be called `default_banner.jpg`.

Reply:
Default should be nothing... not the image that just so happens to be there.

Handle default logic (showing nothing when banner is `None`) in template...